java/arcs/core/storage/StorageProxy.kt

```diff
@@ -380,14 +385,31 @@ class StorageProxy<Data : CrdtData, Op : CrdtOperationAtTime, T>(
         scheduler.schedule(
             MessageFromStoreTask {
                 when (message) {
-                    is ProxyMessage.ModelUpdate -> processModelUpdate(message.model)
+                    is ProxyMessage.ModelUpdate -> {
+                        maybeLogSyncRequestToModelUpdateLatency()
+                        processModelUpdate(message.model)
+                    }
                     is ProxyMessage.Operations -> processModelOps(message.operations)
                     else -> Unit
                 }
             }
         )
     }
 
+    private fun maybeLogSyncRequestToModelUpdateLatency() {
+        analytics?.run {
```

Comment:
Rather than `.run`, I think I prefer `.let`. `.run` isn't super clear that `this` is the receiver of the lambda...

Reply:
OK.